hardware_interface/test/test_component_interfaces.cpp

```diff
@@ -50,9 +50,11 @@ class DummyActuator : public hardware_interface::components::ActuatorInterface
     // We can read a position and a velocity
     std::vector<hardware_interface::StateInterface> state_interfaces;
     state_interfaces.emplace_back(
-      hardware_interface::StateInterface("joint1", "position", &position_state_));
+      hardware_interface::StateInterface(
+        "joint1", hardware_interface::HW_IF_POSITION, &position_state_));
```

Comment:
Looks good to me but I'm a little unhappy how long these names got.

Reply:
I have no idea how to shorten them...

Reply:
Do you have any idea?

Reply:
I like them as-is. My 2cts.